src/routes/embedded/account/account/account.view.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEmbedded } from "~utils/embedded/embedded.hooks";
+import { Link } from "~wallets/router/components/link/Link";
+
+export function AccountEmbeddedView() {
```

Comment:
@Jonathansoufer This screen is only here to access the views to add new wallets, backup shares and export wallets easily after creating an account.

You can probably remove it and just take the users to the extension "home" screen. However, note once you do that change, these 3 screens need to be finished and links to them need to be scattered across the "old" extension UI (e.g. the extension export UI/action should instead take users to the embedded export wallet view).